bin/move-node.js

```diff
@@ -1,16 +1,42 @@
 #!/usr/bin/env node
 
+const [,, toNode, shardMapJson] = process.argv;
+
 const { moveNode, syncShards } = require('../src/move-node');
 const { removeNode } = require('../src/remove-node');
 
+const parseNodeMapping = (input) => {
+  if (!input) {
+    //single node migration with unspecified node
+    return;
+  } else if (input.startsWith('{')) {
```

Comment:
There is no need for an else block here, because you return earlier. Also, I don't really like the checking for json by starts with, I would much rather do something like: 
```javascript
const parseNodeMapping = (input) => {
  if (!input) {
    return;
  }
  try {
    return JSON.parse(input);
  } catch (err) {
    console.warn('Failed to parse node mapping JSON. Defaulting to single node mapping');
    return input;
  }
};
```